src/common/BaseAnnotator.ts

```diff
@@ -37,6 +37,7 @@ export type Options = {
     initialMode?: store.Mode;
     intl: IntlOptions;
     locale?: string;
+    stroke?: Partial<Stroke>;
```

Comment:
This seems like it could spiral out of control somewhat as we add more options. How can we avoid prop sprawl on this top-level constructor?